testRearrangeCarGroup8.py

```diff
@@ -0,0 +1,94 @@
+import unittest
+import numpy as np
+from RearrangeCarGroup8Movements import RearrangeCars
+
+class TestRearrangeCars(unittest.TestCase):

```

Comment:
I cannot see the update but it might be because my git is still connecting to an old version, I'm not sure :/